ui/js/dfv/src/config/prop-types.js

```diff
@@ -89,19 +98,22 @@ export const FIELD_PROP_TYPE_SHAPE = PropTypes.exact( {
 	required: PropTypes.oneOf( BOOLEAN_ALL_TYPES ),
 	roles_allowed: PropTypes.string,
 	sister_id: NUMBER_OR_NUMBER_AS_STRING,
+	slug_placeholder: PropTypes.string,
 	storage_type: PropTypes.string,
 	text_allow_html: PropTypes.oneOf( BOOLEAN_STRINGS ),
 	text_allow_shortcode: PropTypes.oneOf( BOOLEAN_STRINGS ),
 	text_max_length: NUMBER_OR_NUMBER_AS_STRING,
+	text_placeholder: PropTypes.string,
 	text_allowed_html_tags: PropTypes.string,
 	text_repeatable: PropTypes.oneOf( BOOLEAN_STRINGS ),
 	type: PropTypes.string.isRequired,
 	website_format: PropTypes.string,
 	website_allow_port: PropTypes.string,
 	website_clickable: PropTypes.string,
 	website_new_window: PropTypes.string,
-	website_max_length: PropTypes.string,
-	website_html5: PropTypes.string,
+	website_max_length: PropTypes.number,
+	website_html5: PropTypes.oneOf( BOOLEAN_ALL_TYPES ),
+	website_placeholder: PropTypes.string,
```

Comment:
@zrothauser If we add more options in the future, will them not being here be an ongoing problem?

Reply:
I've been trying to add them as I come across new options that aren't here yet - it would be good to keep this list up-to-date, but it won't cause problems if some are missing. There won't be errors if additional properties are included that aren't in this list, but will show a notice in the console for dev builds (everything related to prop types gets stripped out for production builds). And it'll show a warning in the console if one that's marked `isRequired` isn't included. 

Prop types are mostly to provide some basic type checking, and help prevent some bugs from happening: https://reactjs.org/docs/typechecking-with-proptypes.html#gatsby-focus-wrapper